pkg/ccl/spanconfigccl/spanconfigsqltranslatorccl/datadriven_test.go

```diff
@@ -98,6 +117,10 @@ func TestDataDriven(t *testing.T) {
 			tenant = spanConfigTestCluster.InitializeTenant(ctx, roachpb.SystemTenantID)
 		}
 
+		execCfg := tenant.ExecCfg()
```

Comment:
[nit] Remove these intermediate variables, using the symbols on the Tenant handle below directly is no less clear.